tests/unit/client/feedback/dataset/remote/test_dataset.py

```diff
@@ -0,0 +1,251 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from datetime import datetime
+from uuid import uuid4
+
+import httpx
+import pytest
+from argilla import Workspace
```

Comment:
As this is a unit test maybe for consistency we can just rename this import to
```suggestion
from argilla.client.workspaces import Workspace
```

Reply:
Note that the `pre-commit` will fail if you apply the suggestion above, because that import should be placed lower

Reply:
The pre-commit should apply a commit fixing these kinds of things. As I mentioned, I prefer to use Argilla models as a user should do, so it's better to  import as `from argilla import Workspace`